packages/solutions/app-tools/src/plugins/deploy/platforms/node.ts

```diff
@@ -0,0 +1,84 @@
+import path from 'node:path';
+import {
+  ROUTE_SPEC_FILE,
+  DEFAULT_SERVER_CONFIG,
+  fs as fse,
+  getInternalPlugins,
+  chalk,
+} from '@modern-js/utils';
+import { genPluginImportsCode, serverAppContenxtTemplate } from '../utils';
+import { handleDependencies } from '../dependencies';
+import { CreatePreset } from './platform';
+
+export const createNodePreset: CreatePreset = (
+  appContext,
+  config,
+  needModernServer,
+) => {
+  const { appDirectory, distDirectory, serverInternalPlugins } = appContext;
+  const plugins = getInternalPlugins(appDirectory, serverInternalPlugins);
+  const outputDirectory = path.join(appDirectory, '.output');
+  const staticDirectory = path.join(outputDirectory, 'static');
+  const entryFilePath = path.join(outputDirectory, 'index.js');
+  return {
+    async prepare() {
+      await fse.remove(outputDirectory);
+    },
+    async writeOutput() {
+      await fse.copy(distDirectory, outputDirectory);
+    },
+    async genEntry() {
+      if (!needModernServer) {
```

Comment:
now csr project will also print then content in the console....but actually there is no .output/index.js file
![img_v3_02b5_8100b41d-7f00-4fc7-8a08-6eb41c2fbf3g](https://github.com/web-infra-dev/modern.js/assets/68810266/b01c87f4-91f0-43e2-b0b3-cbb7a9610c14)

Reply:
we should fix it in `end()` hook

Reply:
This is expected, when it is a Node.js environment, users needs Modern.js Server to serve the web page.

Reply:
Got it, done, in the CSR, the file is also emitted